### PR TITLE
Fix documentation for the `file-stream` response

### PR DESCRIPTION
**Pull request by bismitpanda**
The newly added `file-stream` response had a few documentation/formatting issues. I tried to keep it as original as possible with minor changes.

### CODE REVIEW

File: axum-extra/src/response/file_stream.rs [hunk 1]
```diff
@@ -22,23 +22,27 @@ use tokio_util::io::ReaderStream;
...
+///     );
+///     let file_stream_resp = FileStream::new(stream).file_name("test.txt");
+///
 ///     Ok(file_stream_resp.into_response())
 /// }
-/// let app = Router::new().route("/FileStreamDownload", get(file_stream));
-/// # let _: Router = app;
+///
+/// let app = Router::new().route("/file-stream", get(file_stream));
+/// let _: Router = app;
```
Comment (Turbo87) [inline axum-extra/src/response/file_stream.rs]:
```suggestion
/// # let _: Router = app;
```

File: axum-extra/src/response/file_stream.rs [hunk 2]
```diff
@@ -65,57 +69,55 @@ where
...
     ///     FileStream::<ReaderStream<File>>::from_path("test.txt")
     ///         .await
     ///         .map_err(|e| (StatusCode::NOT_FOUND, format!("File not found: {e}")))
-    ///         .into_response()
     /// }
-    /// let app = Router::new().route("/FileStreamDownload", get(file_stream));
-    /// # let _: Router = app;
+    ///
+    /// let app = Router::new().route("/file-stream", get(file_stream));
+    /// let _: Router = app;
```
Comment (Turbo87) [inline axum-extra/src/response/file_stream.rs]:
```suggestion
    /// # let _: Router = app;
```

File: axum-extra/src/response/file_stream.rs [hunk 3]
```diff
@@ -136,25 +138,35 @@ where
...
-    ///     let stream=ReaderStream::new(file);
-    ///  
-    ///     Ok(FileStream::new(stream).into_range_response(10, file_size-1, file_size))
+    ///     Ok(FileStream::new(stream).into_range_response(10, file_size - 1, file_size))
     /// }
-    /// let app = Router::new().route("/FileStreamRange", get(range_response));
-    /// # let _: Router = app;
+    ///
+    /// let app = Router::new().route("/file-stream", get(range_response));
+    /// let _: Router = app;
```
Comment (Turbo87) [inline axum-extra/src/response/file_stream.rs]:
```suggestion
    /// # let _: Router = app;
```

**Review by Turbo87 — APPROVED** [review]

**Comment by jplatte** [comment]
Thanks! Wanted to clean this up myself, but completely forgot.

**Pull request merged**